build/move-release-notes

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+const authenticate = require('./github-authentication')
+const { program } = require('commander')
+
+const RELEASE_NOTES_DIR = 'release-notes'
+const UNRELEASED_DIR = 'unreleased'
+
+program
+  .option('-d, --debug', 'enable extreme logging')
+  .requiredOption('--deliverino-private-key <deliverino-private-key>', 'private key of the deliverino github app in pem format with base64 encoding')
+  .requiredOption('--owner <owner>', 'owner of the current repository, usually "Integreat"')
+  .requiredOption('--repo <repo>', 'the current repository, usually "integreat-webapp" or "integreat-react-native-app"')
+  .requiredOption('--branch <branch>', 'the current branch')
+  .requiredOption('--commit-sha <commit-sha>', 'the sha1 of the current commit')
+
+const moveReleaseNotes = async ({ newVersionName, deliverinoPrivateKey, owner, repo, branch, commitSha }) => {
+  const appOctokit = await authenticate({ deliverinoPrivateKey, owner, repo })
+  const lastCommit = await appOctokit.git.getCommit({ owner, repo, commit_sha: commitSha })
+
+  // Tree of the root folder of the project
+  const rootTreeSha = lastCommit.data.tree.sha
+  const rootTree = await appOctokit.git.getTree({ owner, repo, tree_sha: rootTreeSha })
+
+  // Tree of the 'release-notes' folder of the project
+  const releaseNoteTreeSha = rootTree.data.tree.find(it => it.path === RELEASE_NOTES_DIR).sha
+  const releaseNoteTree = await appOctokit.git.getTree({ owner, repo, tree_sha: releaseNoteTreeSha })
+
+  // Rename the dir 'unreleased' to <newVersionName> and keep everything else
+  const renameUnreleasedDir = releaseNoteTree.data.tree.map(it => {
+    const { size, url, ...rest } = it
+    if (it.path === UNRELEASED_DIR) {
+      return { ...rest, path: newVersionName }
```

Comment:
Does this mean the unreleased directory gets removed? Should we put an '.gitkeep' in that folder such that we do not have to recreate this directory every time after a new release?

Reply:
Yes. We could do that, yes. Good suggestion.

Reply:
Hmm quite some more work needed as we have to create a new tree and the .gitkeep every time in the script here. Also we'd probably have to delete it from the new release folder.

The reason for this is that we don't remove the unreleased directory but instead just rename it.

Reply:
Done